packages/block-editor/src/hooks/border.js

```diff
@@ -16,7 +16,41 @@ import { BorderRadiusEdit } from './border-radius';
 import { BorderStyleEdit } from './border-style';
 import { BorderWidthEdit } from './border-width';
 
+const isWeb = Platform.OS === 'web';
+
 export const BORDER_SUPPORT_KEY = '__experimentalBorder';
+export const CSS_UNITS = [
+	{
+		value: 'px',
+		label: isWeb ? 'px' : __( 'Pixels (px)' ),
+		default: '',
+	},
+	{
+		value: 'em',
+		label: isWeb ? 'em' : __( 'Relative to parent font size (em)' ),
+		default: '',
+	},
+	{
+		value: 'rem',
+		label: isWeb ? 'rem' : __( 'Relative to root font size (rem)' ),
+		default: '',
+	},
```

Comment:
If there's a desire to reintroduce max values, maybe here's a decent spot for them?

Reply:
If we could get away with static max values, that would be a good place for them. 

I'm not sure how much sense it makes in the bigger picture. You can get some weird/creative shapes with complete freedom over the border-radius values. That said we generally try to encourage good behaviour by default so I'm by no means against a future iteration to add max values if we have a flexible strategy for them.

Reply:
All good, was asking out of curiosity. Your reasoning make sense to me. 👍

Reply:
Thanks, appreciate the sanity check 👍